ios/bigtable/pom.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
call it `io`, not `ios`.

`bigtable` goes in the Google Cloud Platform module -- `io.gcp.bigtable`?